### DISCUSSION TITLE
Welcome to LokisAsicTester Discussions!

### DISCUSSION THREAD

**Discussion by LOKisGithub** [LOKisGithub/LokisAsicTester#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fdmarcano** [comment]
Hi, i have a problem, when I enter the website a warning "jax error" comes out and does not let me change the config.ini to s9 stays fixed in dr5 every time I applied the changes
